sysinit/coqinit.ml

```diff
@@ -94,7 +94,6 @@ let print_query opts = let open Coqargs in function
     print_endline coqlib
   | PrintHelp h -> Boot.Usage.print_usage stderr h
   | PrintConfig ->
-    let () = init_coqlib opts in
```

Comment:
@Alizter it seems -config was already printing the value as overridden by COQLIB or -coqlib, so this PR does not change this behavior

Reply:
(I removed the call to init_coqlib because the code to print queries is not run after the system is initialized, which includes coqlib initialization).